components/script/dom/htmltextareaelement.rs

```diff
@@ -110,6 +112,9 @@ static DEFAULT_COLS: u32 = 20;
 // https://html.spec.whatwg.org/multipage/#attr-textarea-rows-value
 static DEFAULT_ROWS: u32 = 2;
 
+static DEFAULT_MAX_LENGTH: i32 = -1;
+static DEFAULT_MIN_LENGTH: i32 = -1;
```

Comment:
I'm not sure why these two lines and DEFAULT_ROWS are statics, I believe they really should instead be const.